test/integration/controlplane/transformation/kmsv2_transformation_test.go

```diff
@@ -768,7 +717,7 @@ func assertPodDEKSources(ctx context.Context, t *testing.T, config storagebacken
 
 func TestKMSv2Healthz(t *testing.T) {
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KMSv2, true)()
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KMSv2KDF, randomBool())()
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KMSv2KDF, true)()
```

Comment:
Lets replace the wiring for this to look at an atomic bool that we can swap in tests (but cannot be set by an end user).  Then we can avoid having to make any large changes to our tests.